### PR TITLE
TD-157: rich message icon for rich message

### PR DESCRIPTION
**Pull request by jithinbalu**
### What do you want to achieve?
-> rich message icon for rich message
### How was the code tested?

->
Screen Shot 2019-09-06 at 4 51 11 PM
### What new thing you came across while writing this code? 
->

### Incase you fixed a bug then please describe the root cause of it? 
->

NOTE: Make sure you're comparing your branch with the correct base branch

### CODE REVIEW

File: webplugin/js/app/mck-sidebox-1.0.js [hunk 1]
```diff
@@ -6386,6 +6386,9 @@ var CURRENT_GROUP_DATA={};
...
 
                             emoji_template = result ? emailSvg + result[0]  : emailSvg + '</span><span>Email</span>';
+                        } else if(Kommunicate.isRichTextMessage(message.metadata)) {
+                            emoji_template = '<span class="mck-icon--rich-message"><svg xmlns="http://www.w3.org/2000/svg" width="17" height="17" viewBox="0 0 23 22"><g fill="none" fill-rule="evenodd" opacity=".539" transform="translate(1 1)"><circle cx="6.455" cy="9" r="1" fill="#000" fill-rule="nonzero"/><path stroke="#000" stroke-linecap="round" stroke-linejoin="round" stroke-width="1.6" d="M14.636 16h-1.045l-3.136 4-3.137-4H.763C.342 16 0 15.673 0 15.27V4.36a4.287 4.287 0 0 1 1.356-3.091A4.69 4.69 0 0 1 4.6-.001h15.546c.421 0 .763.328.763.731V16h-6.273z"/><circle cx="10.455" cy="9" r="1" fill="#000" fill-rule="nonzero"/><circle cx="14.455" cy="9" r="1" fill="#000" fill-rule="nonzero"/></g></svg></span><span>'+message.message ||"Message"+'</span>';
```
Comment (devashishmamgain) [inline webplugin/js/app/mck-sidebox-1.0.js]: @jithinbalu pls check if "Message" needs to be from MCK_LABELS?

File: webplugin/js/app/mck-sidebox-1.0.js [hunk 2]
```diff
@@ -6358,7 +6358,7 @@ var CURRENT_GROUP_DATA={};
                 }
             };
             _this.getScriptMessagePreview = function(message,emoji_template){
-				if (message && message.message && message.contentType !== KommunicateConstants.MESSAGE_CONTENT_TYPE.LOCATION && message.contentType !== KommunicateConstants.MESSAGE_CONTENT_TYPE.TEXT_HTML) {
+				if (message && message.message && message.contentType !== KommunicateConstants.MESSAGE_CONTENT_TYPE.LOCATION && message.contentType !== KommunicateConstants.MESSAGE_CONTENT_TYPE.TEXT_HTML && !Kommunicate.isRichTextMessage(message.metadata)) {
```
Comment (CaptainT33mo) [inline webplugin/js/app/mck-sidebox-1.0.js]: @jithinbalu are you sure that `message.metadata` will always be there?

Comment (Suraj-Kumar) [reply webplugin/js/app/mck-sidebox-1.0.js]: Kommunicate.isRichTextMessage has a null check for metadata.

File: webplugin/js/app/mck-sidebox-1.0.js (hunk 1 above)
Comment (Suraj-Kumar) [reply webplugin/js/app/mck-sidebox-1.0.js]: moved message intoMCK_LABELS.

**Pull request merged**